R/utils-add_p.R

```diff
@@ -30,6 +112,13 @@ add_p_test_chisq.test <- function(data, variable, by, ...) {
   result
 }
 
+add_p_test_chisq.test.no.correct <- function(data, variable, by, ...) {
+  result = list()
+  result$p <- stats::chisq.test(data[[variable]], as.factor(data[[by]]), correct = FALSE)$p.value
+  result$test <- "chi-square test of independence"
```

Comment:
the note is added to the table footnote.  i agree it would be more clear with that label, but i think it is a bit overkill to add the information to every table footnote?

Reply:
What do you think @margarethannum ?